### PR TITLE
[Fix] Adding new AdapterDaiMgr and tests.

### PR DESCRIPTION
**Pull request by paul-amonson**
Signed-off-by: Paul Amonson <paul.d.amonson@intel.com>

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -584,7 +584,7 @@ project(':dai_core') {
                 enabled = enforceCodeCoverage
                 limit {
                     counter = 'BRANCH'
-                    minimum = 0.52
+                    minimum = 0.50
```
Comment (sindhu-106) [inline build.gradle:587]: Is this okay ?

**Review by sindhu-106 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (paul-amonson) [reply build.gradle:587]: Yes, I ran it by @nbwhite first. :)

Comment (nbwhite) [reply build.gradle:587]: Confirmed, this is fine, we've got enough wiggle room elsewhere.

**Review by sindhu-106 — DISMISSED** [review]

**Review by sindhu-106 — DISMISSED** [review]

**Comment by paul-amonson** [comment]
Last push addressed the change in MachineConfig.json files.

**Review by sindhu-106 — DISMISSED** [review]

File: config-files/MachineConfig.json [hunk 2]
```diff
@@ -5,49 +5,49 @@
       "ServiceNode":"SN0-SMS1",
```
Comment (nbwhite) [inline config-files/MachineConfig.json:5]: Is this change running up against the security issue we had before with command line invocations?

Comment (paul-amonson) [reply config-files/MachineConfig.json:5]: I am not sure as we are not using the CheckMarx static analysis anymore. We will need to see what Klocwork's says.

**Review by sindhu-106 — APPROVED** [review]

**Review by nbwhite — APPROVED** [review]
Confirmed, this fixes the DAIManager restart bug.

**Pull request merged**